ducktape/tests/message.py

```diff
@@ -0,0 +1,95 @@
+# Copyright 2016 Confluent Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import time
+
+
+class Request(object):
+    READY = "READY"  # reply: {test_metadata, cluster, session_context}
+    SETTING_UP = "SETTING_UP"
+    RUNNING = "RUNNING"
+    TEARING_DOWN = "TEARING_DOWN"
+    FINISHED = "FINISHED"
+    LOG = "LOG"
+
+    def __init__(self, source_id):
```

Comment:
I'm not sure I understand what this class is supposed to represent. From its name I thought it had to do with requests being sent between the different processes, but it seems this object only has a `source_id` and each of the methods seems to generate an `event` more like what I was expecting to be sent back and forth?